CONTRIBUTING.md

```diff
@@ -19,6 +19,21 @@ As contributors and maintainers to this project, you are expected to abide by cu
 conduct. More information can be found at:
 [Contributor Code of Conduct](https://docs.rapids.ai/resources/conduct/).
 
+## Documentation contributions
+
+To make a contribution only to cuDF documentation:
+
+1. Create the `cudf_dev` conda environment following the steps in [Building with a conda environment](#building-with-a-conda-environment).
+2. Activate the environment and install a recent version of `cudf`
+
+```bash
+conda activate cudf_dev
+conda install cudf -c rapidsai-nightly -c conda-forge
+```
+
+3. Build and view the docs locally following the instructions in the [Building
+documentation docs](https://docs.rapids.ai/api/cudf/stable/developer_guide/documentation/#building-and-viewing-docs)
```

Comment:
what do you think about directing the contributor to pick up around step 5 of `code contributions` below after this? The contribution would still need to pass a CI cycle and obtain approvals as normal.

Reply:
To clarify - I think this does a great job of walking a contributor through how to build and view their changes to the docs. Do you think though that its worth noting that after doing these steps, the contributor should proceed with the normal process of creating a PR, obtaining approvals, passing CI, and merging as normal? In particular it might be worth pointing out that passing CI includes a check that will verify the docs build.

Reply:
Yes, that makes total sense, I'll push a fix soon.